README.md

```diff
@@ -34,14 +34,28 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+ A:  The nice thing about semantic HTML is that we can understand it and the computer can understand it. The tags we use have legible meaning for the user as well as functional meaning for the computer.
+
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
 
+A:   'display: block' wants to be alone on a line, while 'display: inline' makes that same block tolerate other blocks being on the same line. 
+
 3. What are the 4 areas of the box model?
 
+
+A: There are the padding edge, content edge, border edge, and margin edge. 
+
+
 4. While using flexbox, what axis does the following property work on: ```align-items: center```?
 
+
+A: The cross axis.
```

Comment:
love the use of cross axis, because the direction change when using flex-direction: row vs. column!